qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -266,6 +266,49 @@ def expectation_value(self, oper, qargs=None):
 
         return pauli_phase
 
+    def equiv(self, other):
+        """Return True if the two generating sets generate the same stabilizer group.
+
+        Args:
+            other (StabilizerState): another StabilizerState.
+
+        Returns:
+            bool: True if other has a generating set that generates the same StabilizerState.
+        """
+        if not isinstance(other, StabilizerState):
+            try:
+                other = StabilizerState(other)
+            except QiskitError:
+                return False
```

Comment:
This is probably better handled by the caller of this function - maybe cut out this conversion?